zerver/tests/test_auth_backends.py

```diff
@@ -21,9 +21,13 @@
 import ujson
 
 class AuthBackendTest(TestCase):
-    def verify_backend(self, backend, good_args={},
-                       good_kwargs={}, bad_kwargs=None,
+    def verify_backend(self, backend, good_args=None,
+                       good_kwargs=None, bad_kwargs=None,
                        email_to_username=None):
+        if good_args is None:
+            good_args = []
```

Comment:
I suppose it could be a tuple?  A dict won't cause it to crash, but it isn't the right semantic option for `*`.  It is for `**`.